src/selection.js

```diff
@@ -71,6 +71,16 @@ var Selection = (function() {
     },
 
     /**
+     * Check if a link can be added.
+     * If the editableHost or a parent is a link element linking
+     * should not be possible.
+     */
+    canLinkBeSet: function() {
+      var linkParents = content.getParentsAndSelf(this.host, 'a');
```

Comment:
Ok. It's still forbidden in HTML5 :)
It's not allowed to place interactive elements inside an `<a>` element:
http://www.w3.org/html/wg/drafts/html/master/single-page.html#the-a-element
http://www.w3.org/html/wg/drafts/html/master/single-page.html#interactive-content-2